bridge/contracts/Bridge.sol

```diff
@@ -51,9 +52,10 @@ contract Bridge is Initializable, IBridge, IERC777Recipient, UpgradablePausable,
     ISideTokenFactory public sideTokenFactory;
     //Bridge_v1 variables
     bool public isUpgrading;
-    uint256 constant public feePercentageDivider = 10000; // Porcentage with up to 2 decimals
+    // Porcentage with up to 2 decimals
```

Comment:
I'd rename to Percentage

Reply:
Done at https://github.com/rsksmart/tokenbridge/pull/242/commits/2e62adf3fcae532f45d9c8c1dd8e57495f771ea3